blazor/query-builder/getting-started.md

```diff
@@ -26,14 +26,14 @@ N> You can also explore our [Blazor QueryBuilder example](https://blazor.syncfus
 
 You can create a **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-## Install Syncfusion Blazor QueryBuilder and Themes NuGet in the App
+## Install Syncfusion Blazor Query Builder and Themes NuGet in the App
 
-To add **Blazor QueryBuilder** component in the app, open the NuGet package manager in Visual Studio (*Tools → NuGet Package Manager → Manage NuGet Packages for Solution*), search and install [Syncfusion.Blazor.QueryBuilder](https://www.nuget.org/packages/Syncfusion.Blazor.QueryBuilder) and [Syncfusion.Blazor.Themes](https://www.nuget.org/packages/Syncfusion.Blazor.Themes/). Alternatively, you can utilize the following package manager command to achieve the same.
+To add **Blazor Query Builder** component in the app, open the NuGet package manager in Visual Studio (*Tools → NuGet Package Manager → Manage NuGet Packages for Solution*), search and install [Syncfusion.Blazor.Query Builder](https://www.nuget.org/packages/Syncfusion.Blazor.Query Builder) and [Syncfusion.Blazor.Themes](https://www.nuget.org/packages/Syncfusion.Blazor.Themes/). Alternatively, you can utilize the following package manager command to achieve the same.
```

Comment:
Revert this
![image](https://github.com/syncfusion-content/blazor-docs/assets/90315799/d260d5ea-2478-4e23-9f79-58c36d2f6c22)

Reply:
reverted